tests/unit/src/main/scala/tests/QuickBuild.scala

```diff
@@ -148,6 +148,7 @@ case class QuickBuild(
         List(
           List(
             "-Yrangepos",
+            "-Ywarn-unused-import", //TODO remove this
```

Comment:
I didn't understand completely how QuickBuild works. Should I create a file next to my test containing the scalacOptions? I saw it was reading `metals.json` under directory target, but this one is generated.

Reply:
If you look at `BaseCodeActionLspSuite` there is:
```scala
  _ <- server.initialize(s"""/metals.json
                                  |{"a":{}}
                                  |/$path
                                  |${input
          .replace("<<", "")
          .replace(">>", "")}
                                  |""".stripMargin)
```

you can add scalacOptions there. It would look something like:
```scala
  _ <- server.initialize(s"""/metals.json
                                  |{"a":{"scalacOptions": [""-Ywarn-unused-import""]}}
                                  |/$path
                                  |${input
          .replace("<<", "")
          .replace(">>", "")}
                                  |""".stripMargin)
```

Reply:
ah !! thanks